### PR TITLE
Travis CI: Test current Python versions on focal

### PR DESCRIPTION
**Pull request by cclauss**
Output: https://travis-ci.org/github/aboSamoor/polyglot/pull_requests

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,19 +1,27 @@
...
   - "3.6"
   - "3.5"
-  - "3.4"
   - "2.7"
-#  - "pypy"
+# - "pypy3"
+
+jobs:
+  allow_failures:
+    - python: "3.9-dev"
```
Comment (aboSamoor) [inline .travis.yml]: why test against this if we will allow it to fail. Maybe stopping at 3.8 is good enough for now.

Comment (cclauss) [reply .travis.yml]: The Travis CI documentation around allow_failures are quite clear about when and why it should be used.

Python 3.9 releases in 10 days.

**Comment by cclauss** [comment]
@aboSamoor Can you please visit https://travis-ci.org/github/aboSamoor/polyglot and at the upper-right click "More actions" and then "Trigger build" on master.  This should get the Travis reconnected to GitHub.

File: .travis.yml (hunk 1 above)
Comment (aboSamoor) [reply .travis.yml]: any further changes before i merge this PR?

Comment (cclauss) [reply .travis.yml]: No further changes.  Thanks.

**Pull request closed without merging**